_data/faq.yml

```diff
@@ -1,3 +1,131 @@
+- id: design
+  title:
+      en: Recent changes to the c:geo design
+      de: Kürzliche Design-Änderungen in c:geo
+  items:
+      - id: design-generic
+        en:
+            title: Why did you change the c:geo design?
+            content: |
+                Major parts of c:geo have been designed many years ago and were targeted to very old Android versions. Some of those designs have not even been touched ever since.
+                Over the last years much effort was needed to bugfix and patch this really old design to make it usable for new screens and features.
+                
+                Finally we ended up with a very unstructured design, which made c:geo look inconsistent or even broken at many places.
+                Additionally the old design did not allow us to make use of modern app design principles and features, which you know from many other Android apps.
+                
+                Therefore we finally decided to start from scratch and implement the so-called material design into c:geo. 
+                We tried our best to implement it in a way, that you will still recognize c:geo but have a more consistent and structured appearance.
+                
+                At some places this is still work in progress and we are unable to test it on all the different Android devices and versions.
+                Thus there might still be some minor bugs. Feel free to provide <a href="mailto:support@cgeo.org?subject=New design">feedback to our support</a> if you encounter problems or have improvement proposals.
+        de:
+            title: Warum wurde das c:geo Design geändert?
+            content: |
+                TBC     
+      - id: design-background
+        en:
+            title: Where is my background image?
+            content: |
+                The transparent background was a rather old feature and is no longer used in modern designs. Therefore this feature has been removed.
+                
+                We are planning to change the main menu as well in future versions and migrate to a bottom navigation with a dashboard. 
+                It might be possible, that such dashboard will bring back a similar customization option. We are still working on it.
```

Comment:
Although I understand the argument to smooth the users, it looks a bit inconsistent to me: Above you say "it's an old feature no longer used in modern design" - and then follow-up with "but we may bring it back". I would leave out the last line or at least the first sentence in the last line.

Reply:
Fair enough. Will adapt.